index.html

```diff
@@ -277,13 +176,13 @@ <h2>Skills</h2>
 <footer id="footer">
     <section class="contact-me">
         <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
-        <form action="https://formspree.io/f/xpzbjqge" method="post">
-            <input class="name" type="text" name="Name" maxlength="30" placeholder="Full name" required>
+        <form action="https://formspree.io/f/xpzbjqge" method="post" id="contact-register">
+            <input id="name" type="text" name="Name" maxlength="30" placeholder="Full name" required>
             <hr>
-            <input class="email" type="email" name="Email" placeholder="Email address" required
-                pattern="[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,6}$">
+            <input id="email" type="email" name="Email" placeholder="Email address" required
+                pattern="[a-z0-9._%+-]+[@a-z0-9.-]+\.[a-z]{2,6}$">
             <hr>
-            <textarea class="textarea" name="Message" cols="30" maxlength="500" rows="10"
+            <textarea id="textarea" name="Message" cols="30" maxlength="500" rows="10"
                 placeholder="Enter text here" required></textarea>
```

Comment:
kindly this section should be worked in your JS file. and the email should be required in lowercase. and for the validation, you should sue the mailRegax. please initialize like `const mailRegex = /^[a-z][a-z0-9\-_.]+@[a-z]{2,}\.[a-z0-9]{2,}$/;` you create a function that contains the email validation log. don't forget to declare your ids in your js file